### ISSUE TITLE
ULS UNITS ASIA - Logo text needs to meet ratio

### ISSUE THREAD

**Issue by kevinkidwell** [cul-it/uls#456]
Warning - Logo image of text fails 3:1 against the header.  Branding is exempt from color contrast but this may or may not pass as branding.
